pkgs/development/python-modules/pip-chill/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  pip,
+  pythonAtLeast,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "pip-chill";
+  version = "1.0.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "rbanffy";
+    repo = "pip-chill";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-oWq3UWBL5nsCBUkaElashZKvm7pN3StJNubgU++8YFs=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = lib.optionals (pythonAtLeast "3.12") [ setuptools ];
+
+  nativeCheckInputs = [
+    pip
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "pip_chill" ];
+
+  preCheck = ''
+    substituteInPlace tests/test_pip_chill.py \
+      --replace-fail "pip_chill/cli.py" "${placeholder "out"}/bin/pip-chill"
+  '';
```

Comment:
```
 preCheck = ''
    export PATH=$out/bin:$PATH
  '';
```
Would this work?

Reply:
Yes it works

Reply:
Seems a better version then, smaller and uncoupled from specific files.

Reply:
Yeah, I didn't thought of this

Reply:
> Yeah, I didn't thought of this

Neither did I. The nixpkgs collective is as good as the [Borg](https://en.wikipedia.org/wiki/Borg). Resistance is futile.